docs/core/diagnostics/index.md

```diff
@@ -79,6 +79,10 @@ The [dotnet-gcdump](dotnet-gcdump.md) tool is a way to collect GC (Garbage Colle
 
 [Tutorial: Debug deadlock](debug-deadlock.md) shows you how to use the [dotnet-dump](dotnet-dump.md) tool to investigate threads and locks.
 
+### Debug a StackOverflow
+
+[Tutorial: Debug a StackOverflow](debug-stackoverflow.md) demonstrates how to debug a <xref:System.StackOverflowException> on Linux.
+
 ### Debug Linux dumps
 
```

Comment:
Can we also add this in toc.yml? Should be under docs/fundamentals/toc.yml

Reply:
Done